parent/pom.xml

```diff
@@ -173,7 +173,7 @@
       <version.weld-se>2.2.0.SP1</version.weld-se>
 
       <version.org.wildfly>10.0.0.CR5</version.org.wildfly>
-      <version.org.wildfly.core>2.0.5.Final</version.org.wildfly.core>
+      <version.org.wildfly.core>2.0.7.Final</version.org.wildfly.core>  <!-- mstodo undo ? -->
```

Comment:
We don't need it. I will undo the change - I used it for debugging locally (with the old one I had two versions of libs)